docs/operator-manual/upgrading/2.3-2.4.md

```diff
@@ -69,20 +69,77 @@ server's key signature algorithms configuration.
 > authentication via the HostkeyAlgorithms and PubkeyAcceptedAlgorithms
 > options. For example, the following stanza in ~/.ssh/config will enable
 > RSA/SHA1 for host and user authentication for a single destination host:
-> 
-> ```
-> Host old-host
->     HostkeyAlgorithms +ssh-rsa
->     PubkeyAcceptedAlgorithms +ssh-rsa
-> ```
-> 
+ 
+ ```
+ Host old-host
+     HostkeyAlgorithms +ssh-rsa
+     PubkeyAcceptedAlgorithms +ssh-rsa
+ ```
+ 
```

Comment:
@34fathombelow is the formatting getting messed up in readthedocs? Would be nice to keep it in the block quote, since it's from the release notes. But understandable if formatting is broken.

Reply:
Yeah i tried to keep it in the block.   The formatting was messed up.  I could not find another solution